arviz/plots/compareplot.py

```diff
@@ -81,7 +81,7 @@ def plot_compare(
 
     Returns
     -------
-    axes : matplotlib axes or bokeh figures
+    axes : matplotlib _axes or bokeh _figure

```

Comment:
```suggestion
    axes : matplotlib_axes or bokeh_figure
```

same comment as above, extra spaces within the aliases which will break the rendering.